ChineseFilter/sections/specific.txt

```diff
@@ -956,3 +956,4 @@ nfmovies.com##body div[id^="aaaDiv"]
 ||nfmovies.com/static/side.jpg$image,redirect=nooptext,domain=nfmovies.com
 ! https://github.com/AdguardTeam/AdguardFilters/pull/90035
 ||pgdt.gtimg.cn/*/snscosdownload/*/reserved/$app=com.tencent.qqmusic
+||pgdt.gtimg.cn/gdt/$image,$app=com.tencent.qqmusic
```

Comment:
`$app=` => `app=`

Reply:
Hello!
OK, the changes are done, thank you